client/components/LearnerViewDirective.js

```diff
@@ -96,12 +96,24 @@ tie.directive('learnerView', [function() {
               <h3 class="tie-question-title">{{title}}</h3>
               <div class="tie-previous-instructions">
                 <div ng-repeat="previousInstruction in previousInstructions track by $index">
-                  <p ng-repeat="paragraph in previousInstruction track by $index">{{paragraph}}</p>
+                  <div ng-repeat="instruction in previousInstruction track by $index">
+                    <p ng-if="instruction.type == 'text'">{{instruction.content}}</p>
+                    <div ng-if="instruction.type == 'example'">
+                      Example: <pre class="tie-question-example">{{instruction.content}}</pre>
```

Comment:
I think we should be more flexible in what uses <pre> format. Meaning, remove the 'Example: ' in the line above.

Also means we probably want to name this type something more general than 'example'. Maybe 'preformatted' or 'codetext'